packages/web/app/edge/images/route.ts

```
@@ -124,15 +127,18 @@ export async function GET(request: NextRequest) {
 
     const url = searchParams.get('url')
     const width = searchParams.get('w')
-      ? parseInt(searchParams.get('w')!)
+      ? parseInt(searchParams.get('w') || '0')
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)